reference/config_files/global_conf.rst

```diff
@@ -31,6 +31,34 @@ To list all the possible configurations available, run :command:`conan config li
 
 .. include:: ../../common/config_list.inc
 
+Description of configurations
++++++++++++++++++++++++++++++
+
+core.cache:storage_path
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Absolute path to a folder where the Conan packages and the database of the packages will be stored.
+This folder will take the most of the space in your disk, as it stores the binary pacakges dowloaded or created.
```

Comment:
```suggestion
This folder will take most of the space in your disk if left unchecked, as it stores both the binary packages dowloaded or created.
```

Reply:
what do you mean by that "if left unchecked"?

Reply:
just to clarify, I wanted to say that this folder will be the heaviest conan storage folder. Not that this will take the space of the user's disk for sure 😄